templates/vue/src/components/TapestryApp.vue

```diff
@@ -32,13 +47,26 @@ export default {
     }
   },
   computed: {
-    ...mapState(["nodes", "links", "selection", "settings", "rootId"]),
+    ...mapState(["nodes", "links", "selection", "settings", "rootId", "avatar"]),
     isSidebarOpen() {
       return Boolean(this.$route.query.sidebar)
     },
     analyticsEnabled() {
       return this.settings.analyticsEnabled
     },
+    hasAvatar() {
+      return !(!this.avatar || Object.keys(this.avatar).length === 0)
+    },
+    avatarPopup() {
+      if (
+        process.env.VUE_APP_AVATAR_POPUP === "true" ||
+        process.env.VUE_APP_AVATAR_POPUP === "TRUE" ||
+        process.env.VUE_APP_AVATAR_POPUP === 1
+      ) {
```

Comment:
I believe there is no need for three "or" as JS has a spacial definition for "truly" expressions - https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
Good catch, thanks!